src/main/java/com/google/codeu/data/Message.java

```diff
@@ -41,6 +41,19 @@ public Message(UUID id, String user, String text, long timestamp) {
     this.timestamp = timestamp;
   }
 
+  public Message(String user, String text, String location) {
+    this(UUID.randomUUID(), user, text, System.currentTimeMillis());
```

Comment:
Nit: Following the existing code, this could be done in one line using the new constructor.
```suggestion
    this(UUID.randomUUID(), user, text, System.currentTimeMillis(), location);
```